packages/backend/server/src/plugins/copilot/providers/openai.ts

```diff
@@ -246,10 +246,12 @@ export class OpenAIProvider
 
       const [system, msgs] = await chatToGPTMessage(messages);
 
-      const modelInstance = this.#instance(model, {
-        structuredOutputs: Boolean(options.jsonMode),
-        user: options.user,
-      });
+      const modelInstance = options.webSearch
+        ? this.#instance.responses(model)
+        : this.#instance(model, {
+            structuredOutputs: Boolean(options.jsonMode),
+            user: options.user,
+          });
 
```

Comment:
Why `structuredOutputs` is related to `webSearch` config?

Reply:
responses api not need this option

Reply:
At least change the code to：
```
options. jsonMode
        ? this.#instance(model, {
            structuredOutputs: Boolean(options.jsonMode),
            user: options.user,
          }): this.#instance.responses(model);
```